ide/libs.graalsdk/test/unit/src/org/netbeans/libs/graalsdk/GraalEnginesTest.java

```diff
@@ -70,7 +70,7 @@ public void invokeEngineViaGeneratedScriptEngine() {
                 .append("\nnames: ").append(engineNames)
                 .append("\ntypes: ").append(types);
 
-            if (engineNames.contains("llvm")) {
+            if (engineNames.contains("LLVM")) {
```

Comment:
Wouldn't it be safer to ignore character case ? Engine names are case-sensitive, though.

Reply:
Looks like the name changed between RC10 to RC12 to upper case. 

I was thinking of doing `toLowerCase()`. On the other hand, it is just a test. If it changes again with subsequent RC, we update the test.

Reply:
Good point; on a second thought, if the shortName changes after `1.0.0`, file a bug against GraalVM as well, since the token can be used in `ScriptEngineManager.getEngineByName()`, it would be an incompatible change. So it's good that the test would fail in this case.